packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -363,6 +358,7 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
     _resolutionPreset = getResolutionPresetForString(resolutionPreset);
   } @catch (NSError *e) {
     *error = e;
+    return nil;
   }
```

Comment:
@mvanbeusekom and @stuartmorgan If this doesn't return nil setting the error is useless. I assume this doesn't give real-life issues as the resolutionPreset is always set to something valid. Can I just include the return in this PR or should I make a separate issue and PR for it?

Reply:
Chose to exclude it from this PR